pkg/storage/system_intake_grb_reviewer.go

```diff
@@ -0,0 +1,68 @@
+package storage
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/uuid"
+	"github.com/jmoiron/sqlx"
+	"github.com/lib/pq"
+	"go.uber.org/zap"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/models"
+	"github.com/cmsgov/easi-app/pkg/sqlqueries"
+)
+
+// CreateSystemIntakeGRBReviewer creates a GRB Reviewer
+func (s *Store) CreateSystemIntakeGRBReviewer(ctx context.Context, tx *sqlx.Tx, systemIntakeID uuid.UUID, reviewer *models.SystemIntakeGRBReviewer) error {
+
+	if reviewer.ID == uuid.Nil {
+		reviewer.ID = uuid.New()
+	}
+	if _, err := tx.NamedExec(sqlqueries.SystemIntakeGRBReviewer.Create, reviewer); err != nil {
+		appcontext.ZLogger(ctx).Error("failed to create GRB reviewer", zap.Error(err))
+		return err
+	}
+
+	return nil
+}
+
+func (s *Store) UpdateSystemIntakeGRBReviewer(ctx context.Context, tx *sqlx.Tx, reviewerID uuid.UUID, votingRole models.SystemIntakeGRBReviewerVotingRole, grbRole models.SystemIntakeGRBReviewerRole) (*models.SystemIntakeGRBReviewer, error) {
+	var reviewer models.SystemIntakeGRBReviewer
+	stmt, err := tx.PrepareNamed(sqlqueries.SystemIntakeGRBReviewer.Update)
+	if err != nil {
+		appcontext.ZLogger(ctx).Error("failed to prepare update GRB reviewer query", zap.Error(err))
+		return nil, err
+	}
+	defer stmt.Close()
+	err = stmt.Get(&reviewer, args{
+		"reviewer_id": reviewerID,
+		"grb_role":    grbRole,
+		"voting_role": votingRole,
+		"modified_by": appcontext.Principal(ctx).Account().ID,
+	})
+	fmt.Println(reviewer)
```

Comment:
✍️

Reply:
Addressed in [889b12f](https://github.com/CMSgov/easi-app/pull/2646/commits/889b12f9d8bf02015fdb2660a0b5316aaf1da555)